build/grunt.js

```diff
@@ -325,11 +327,30 @@ module.exports = function(grunt) {
         }
       })
     },
+    babel: {
+      options: {
+        sourceMap: true,
+        presets: ['es2015'],
```

Comment:
Yes, it should probably be blank. I think the issue was that other things in the build pipeline wanted sourceMaps from babel. I'll take a look at this.